PIDs:
  X:
    kp: 0.2
    kd: 0.0
    ki: 0.0
    minOutput: -1.0
    maxOutput: 1.0
    integratorMin: -0.1
    integratorMax: 0.1
  Y:
    kp: 0.2
    kd: 0.0
    ki: 0.0
    minOutput: -1.0
    maxOutput: 1.0
    integratorMin: -0.1
    integratorMax: 0.1
  Z:
    kp: 0.2
    kd: 0.0
    ki: 0.0
    minOutput: -1.0
    maxOutput: 1.0
    integratorMin: -0.1
    integratorMax: 0.1
  Yaw:
    kp: 0.2
    kd: 0.0
    ki: 0.0
    minOutput: -1.0
    maxOutput: 1.0
    integratorMin: -0.1
    integratorMax: 0.1
  StabilizeTime: 5.0
